sdk/core/core-lro/src/lroEngine/requestUtils.ts

```diff
@@ -41,7 +41,7 @@ export function inferLroMode(
       resourceLocation:
         requestMethod === "PUT"
           ? requestPath
-          : requestMethod === "POST"
+          : ["POST", "PATCH"].includes(requestMethod)
```

Comment:
Nit: not sure if it makes much of a perf difference, but for two entries `[].includes` feels a little like overkill

```suggestion
          : (requestMethod === "POST" || requestMethod === "PATCH")
```